src/modules/anim_controller.cpp

```diff
@@ -358,4 +374,23 @@ namespace Modules::AnimController
 			NRF_LOG_INFO("StartTime %d, remapFace %d, loop %d", anim->startTime, anim->remapFace, anim->loop ? 1: 0);
 		}
 	}
+
+	/// <summary>
+	/// Method used by clients to request timer callbacks
+	/// </summary>
+	void hook(AnimControllerClientMethod method, void* parameter)
+	{
+		if (!clients.Register(parameter, method))
+		{
+			NRF_LOG_ERROR("Too many animation controller hooks registered.");
```

Comment:
Update message to be more specific, like "Failed to register AnimationController hook because [...]"